### ISSUE TITLE
Respawn of an icons' size past problem

### ISSUE THREAD

**Issue by Jerome-Herbinet** [nextcloud/server#38616]
@nextcloud/designers 

If I remember well, I solved this in the past, but it appears again ... and the same CSS code is responsible. 

Removing the 44px width and height may fix the problem but, before making a PR, can someone tell me it won't generate side effect ? Are those properties used elsewhere ?

**Concerned code :** https://github.com/nextcloud/server/blob/09c5f997c6d185d8b23b37a996e7a1130a426d75/core/css/apps.scss#LL1224C1-L1238C1

**Screenshot :** 

![2023-06-02_16-36](https://github.com/nextcloud/server/assets/33763786/9dc69ec0-91a0-4c2a-871a-b7ceaffcc773)

**Comment by szaimen** [comment]
I cannot reproduce this on nc 27.0.0 RC3. Did you already clear the browser cache?

**Comment by Jerome-Herbinet** [comment]
> I cannot reproduce this on nc 27.0.0 RC3. Did you already clear the browser cache?

I use the last minor version of NC 26, but I can see that there is no problem in NC 27 
Could this fix be backported to NC 26 ?

**Comment by szaimen** [comment]
You are right. On Nc26.0.2 I can reproduce this. The question is now if this got broken again or if we forgot to backport a PR or something...

**Comment by nextcloud-command** [comment]
This issue has been automatically marked as stale because it has not had recent activity and seems to be missing some essential information. It will be closed if no further activity occurs. Thank you for your contributions.

**Comment by Jerome-Herbinet** [comment]
> You are right. On Nc26.0.2 I can reproduce this. The question is now if this got broken again or if we forgot to backport a PR or something...

@szaimen any news about concerning a fix for this issue (which has been automatically closed) ?

**Comment by Jerome-Herbinet** [comment]
Seems to be fixed in NC Hub 6
